src/components/home/Socials.jsx

```diff
@@ -11,7 +11,7 @@ const Socials = () => {
           {socials.map((social, index) => (
```

Comment:
use motion.div to have have the appear on view just like in the projects card on the website